node/cli/src/cli.rs

```diff
@@ -222,7 +222,20 @@ impl RelayChainCli {
 		para_config: &sc_service::Configuration,
 		relay_chain_args: impl Iterator<Item = &'a String>,
 	) -> Self {
-		let extension = chain_spec::Extensions::try_get(&*para_config.chain_spec);
+		let extension = {
+			#[cfg(feature = "with-moonbase-runtime")]
+			{
+				chain_spec::moonbase::Extensions::try_get(&*para_config.chain_spec)
+			}
+			#[cfg(feature = "with-moonbeam-runtime")]
+			{
+				chain_spec::moonbeam::Extensions::try_get(&*para_config.chain_spec)
+			}
+			#[cfg(feature = "with-moonriver-runtime")]
+			{
+				chain_spec::moonriver::Extensions::try_get(&*para_config.chain_spec)
+			}
+		};
```

Comment:
Is the idea that each build will include exactly one runtime? Or is it possible to use multiple of them together?

If we want to support multiple, then I believe this won't compile.